extensions/azdata/src/azdata.ts

```diff
@@ -80,17 +77,18 @@ export async function downloadAndInstallAzdata(outputChannel: vscode.OutputChann
  * @param outputChannel Channel used to display diagnostic information
  */
 async function downloadAndInstallAzdataWin32(outputChannel: vscode.OutputChannel): Promise<void> {
-	const downloadPath = path.join(os.tmpdir(), `azdata-msi-${uuid.v4()}.msi`);
-	outputChannel.appendLine(loc.downloadingTo('azdata-cli.msi', downloadPath));
-	await HttpClient.download(`${azdataHostname}/${azdataUri}`, downloadPath, outputChannel);
-	await executeCommand('msiexec', ['/i', downloadPath], outputChannel);
+	const downloadPath = path.join(os.tmpdir());
+	const downloadedFile = await HttpClient.download(`${azdataHostname}/${azdataUri}`, downloadPath, outputChannel);
+	await executeCommand('msiexec', ['/qn', '/i', downloadedFile], outputChannel);
 }
 
 /**
  * Runs commands to install azdata on MacOS
  */
-async function installAzdataDarwin(): Promise<void> {
-	throw new Error('Not yet implemented');
+async function installAzdataDarwin(outputChannel: vscode.OutputChannel): Promise<void> {
+	await executeCommand('brew', ['tap', 'microsoft/azdata-cli-release'], outputChannel);
```

Comment:
do handle different prerelease version of azdata, this may need to be different. What are you thoughts on handling those scenarios? Do you think we will just ship different version of this extension one for appropriate release vehicle?

Reply:
Not doing anything special for prerelease at this time. Users will just be instructed to install azdata ahead of time.